services/resource-usage-tracker/tests/unit/with_dbs/test_process_rabbitmq_message_triggered_by_listening.py

```diff
@@ -0,0 +1,54 @@
+from datetime import datetime, timezone
+from typing import Callable
+
+import sqlalchemy as sa
+from models_library.rabbitmq_messages import (
+    RabbitResourceTrackingBaseMessage,
+    RabbitResourceTrackingHeartbeatMessage,
+    RabbitResourceTrackingStoppedMessage,
+    SimcorePlatformStatus,
+)
+from servicelib.rabbitmq import RabbitMQClient
+
+from .conftest import assert_service_runs_db_row
+
+pytest_simcore_core_services_selection = [
+    "postgres",
+    "rabbit",
+]
+pytest_simcore_ops_services_selection = [
+    "adminer",
+]
+
+
+# NOTE: This test fails when running locally and you are connected through VPN: Temporary failure in name resolution [Errno -3]
```

Comment:
Why can't we do soemthing about this?

Reply:
It works in CI and also locally. By coincidence, I found out that it is not working when you are developing locally but connected through VPN. This is a comment about that here.